src/Sarif/NotYetAutoGenerated/SarifRewritingVisitor.cs

```diff
@@ -416,14 +416,6 @@ public virtual Invocation VisitInvocation(Invocation node)
                     }
                 }
 
-                if (node.Attachments != null)
```

Comment:
finally learnt about NotYetGenerated (the hard way :/)

Reply:
nice!

---
In reply to: [259196449](https://github.com/Microsoft/sarif-sdk/pull/1305#discussion_r259196449) [](ancestors = 259196449)

Reply:
+1

---
In reply to: [259459661](https://github.com/Microsoft/sarif-sdk/pull/1305#discussion_r259459661) [](ancestors = 259459661,259196449)